README.md

```diff
@@ -1,4 +1,4 @@
 # maven2gradle
 converts maven dependencies to gradle format
 
-**check it out in [here](http://sagioto.github.io/maven2gradle/)**
+**check it out in [here](https://evandrojr.github.io/maven2gradle/)**
```

Comment:
You should leave original repo path

Reply:
Sorry it was not supposed to go in the PR